### DISCUSSION TITLE
Expand search query results?

### DISCUSSION THREAD

**Discussion by NGUYENTRONGDAT123** [pystardust/ani-cli#780] [Q&A] [answered]
Hey, when I search for anime, for example, 'Kimetsu no Yaiba', it only shows 6 results in the terminal whereas there should be more than that. How can I display all the results relating to that anime?

here is how many different versions of `Kimetsu no Yaiba` in `animixplay`.
![image](https://user-images.githubusercontent.com/49813515/174543754-db032aa8-c350-4d33-a25e-b57a31c9e0ca.png)

here is the results when I'm trying to find it on my terminal:
![image](https://user-images.githubusercontent.com/49813515/174543418-6e0b9efa-08fb-4af8-b607-f36b1bdcdcea.png)

**Comment by CoolnsX** [comment, marked as answer]
Already fixed in latest version..
